tests/dialects/test_snowflake.py

```diff
@@ -832,6 +832,18 @@ def test_snowflake(self):
                 "snowflake": "SELECT LISTAGG(col1, ', ') WITHIN GROUP (ORDER BY col2) FROM t",
             },
         )
+        self.validate_all(
+            "SELECT APPROX_PERCENTILE(a, 0.5) FROM t",
+            read={
+                "trino": "SELECT APPROX_PERCENTILE(a, 1, 0.5, 0.001) FROM t",
+                "presto": "SELECT APPROX_PERCENTILE(a, 1, 0.5, 0.001) FROM t",
+            },
```

Comment:
Can we use `self.unsupported` to note notify the user that `0.5` and `0.001` couldn't be transpiled? I _think_ these correspond to Presto's:

```
approx_percentile(x, w, percentage, accuracy) → [same as x]
```

case, and Trino simply supports it due to inheritance. @baruchoxman can you verify that this is the correct / intended prototype?

Reply:
Yes that seems correct, that's what I remember too.

Reply:
I wonder if it can be possible to override this behavior, and ignore the weight given that the weight is 1 (and thus can be ignored)?
And same for the accuracy parameter - just ignore it when transpiling to Snowflake. 
I guess we can always create our derived Snowflake dialect which overrides the `approxquantile_sql` funciton behavior, but given that this is the format certain BI tools create in their Trino queries, wondering if can have a default